lambdas/build-proven-user-identity-details/src/test/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandlerTest.java

```diff
@@ -103,27 +100,22 @@ void setUp() {
 
     @Test
     void shouldReceive200ResponseCodeProvenUserIdentityDetails() throws Exception {
-        when(mockUserIdentityService.isVcSuccessful(any(), any())).thenCallRealMethod();
+        when(mockUserIdentityService.isVcSuccessful(any(), any())).thenReturn(true);
         when(mockUserIdentityService.findIdentityClaim(any())).thenCallRealMethod();
         when(mockIpvSessionService.getIpvSession(SESSION_ID)).thenReturn(mockIpvSessionItem);
-        when(mockVerifiableCredentialService.getVcStoreItems(TEST_USER_ID))
+        when(VcHelper.isSuccessfulVc(any()))
+                .thenReturn(true)
+                .thenReturn(true)
+                .thenReturn(true)
+                .thenReturn(true)
+                .thenReturn(true);
```

Comment:
Do you actually need to specify this repeatedly? I think by default the last one applies to all subsequent calls

If you need to specify different return values for repeated calls, you can also do `.thenReturn(true, true, true...)` if you prefer.